modules/knative-eventing-CR-system-deployments.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: REFERENCE
+[id="knative-eventing-CR-system-deployments_{context}"]
+= Overriding Knative Eventing system deployment configurations
```

Comment:
This is titled like it's a procedure module, but marked as a reference. Since it does seem to kind of tell you how to do something, maybe consider whether it should be made into a procedure module with steps?

Edit to say that I see it was based off of another module that was done similarly. So understand keeping it as is for consistency, but maybe something to consider in the future.